pkg/logger/logger.go

```diff
@@ -0,0 +1,102 @@
+package logger
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/rs/zerolog"
+)
+
+// Interface -.
+type Interface interface {
+	Debug(message interface{}, args ...interface{})
+	Info(message string, args ...interface{})
+	Warn(message string, args ...interface{})
+	Error(message interface{}, args ...interface{})
+	Fatal(message interface{}, args ...interface{})
+}
+
+// Logger -.
+type Logger struct {
+	logger *zerolog.Logger
+}
+
+var _ Interface = (*Logger)(nil)
+
+// New -.
+func New(level string) *Logger {
+	var l zerolog.Level
+
+	switch strings.ToLower(level) {
+	case "error":
+		l = zerolog.ErrorLevel
+	case "warn":
+		l = zerolog.WarnLevel
+	case "info":
+		l = zerolog.InfoLevel
+	case "debug":
+		l = zerolog.DebugLevel
+	default:
+		l = zerolog.InfoLevel
+	}
+
+	zerolog.SetGlobalLevel(l)
+
+	skipFrameCount := 2
+	logger := zerolog.New(os.Stdout).With().Timestamp().CallerWithSkipFrameCount(zerolog.CallerSkipFrameCount + skipFrameCount).Logger()
+
+	return &Logger{
+		logger: &logger,
+	}
+}
+
+// Debug -.
+func (l *Logger) Debug(message interface{}, args ...interface{}) {
+	l.msg("debug", message, args...)
+}
+
+// Info -.
+func (l *Logger) Info(message string, args ...interface{}) {
+	l.log(message, args...)
+}
+
+// Warn -.
+func (l *Logger) Warn(message string, args ...interface{}) {
+	l.log(message, args...)
+}
+
+// Error -.
+func (l *Logger) Error(message interface{}, args ...interface{}) {
+	if l.logger.GetLevel() == zerolog.DebugLevel {
+		l.Debug(message, args...)
+	}
+
+	l.msg("error", message, args...)
+}
+
+// Fatal -.
+func (l *Logger) Fatal(message interface{}, args ...interface{}) {
+	l.msg("fatal", message, args...)
+
+	os.Exit(1)
+}
+
+func (l *Logger) log(message string, args ...interface{}) {
+	if len(args) == 0 {
+		l.logger.Info().Msg(message)
+	} else {
+		l.logger.Info().Msgf(message, args...)
+	}
```

Comment:
по идее должно быть достаточно
```suggestion
	l.logger.Info().Msgf(message, args...)
```

Reply:
если args пустой, то zerologer должен его воспринять норм. или я не прав?